src/api/Jalasoft.TeamUp.Resumes.API/Controllers/GetSkillsByName.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace Jalasoft.TeamUp.Resumes.API.Controllers
+{
+    using System;
+    using System.Net;
+    using Jalasoft.TeamUp.Resumes.Core.Interfaces;
+    using Jalasoft.TeamUp.Resumes.Models;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Azure.WebJobs.Extensions.Http;
+    using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Attributes;
+    using Microsoft.Extensions.Primitives;
+    using Microsoft.OpenApi.Models;
+
+    public class GetSkillsByName
+    {
+        private readonly ISkillsService skillsService;
+
+        public GetSkillsByName(ISkillsService skillsService)
+        {
+            this.skillsService = skillsService;
+        }
+
+        [FunctionName("GetSkillsByName")]
+        [OpenApiOperation(operationId: "GetSkillsByName", tags: new[] { "Skills" })]
+        [OpenApiParameter(name: "name", In = ParameterLocation.Query, Required = true, Type = typeof(string), Description = "The skill name.")]
+        [OpenApiResponseWithBody(statusCode: HttpStatusCode.OK, contentType: "application/json", bodyType: typeof(Data), Description = "Successful response")]
```

Comment:
You should get the name through params for that you need to change In = ParameterLocation.Query to In = ParameterLocation.Path